src/sentry/static/sentry/app/views/stream/tagFilter.jsx

```diff
@@ -71,14 +73,24 @@ class StreamTagFilter extends React.Component {
     this.api
       .requestPromise(this.getTagValuesAPIEndpoint(), {
         query: {
-          query: this.state.textValue,
+          query: textValue,
         },
       })
       .then(resp => {
         this.setState({
           isLoading: false,
           options: Object.values(resp).map(StreamTagFilter.tagValueToSelectFormat),
         });
+      })
+      .catch(err => {
+        // TODO(billy): This endpoint seems to timeout a lot,
+        // should we log these errors into datadog?
```

Comment:
Are we able to make the endpoint faster? Or is this one of the endpoints that gets better with snuba?